src/User32.Desktop/User32.cs

```diff
@@ -144,6 +159,12 @@ public static partial class User32
         /// </returns>
         public delegate IntPtr DialogProc(IntPtr hwndDlg, WindowMessage uMsg, IntPtr wParam, IntPtr lParam);
 
+        public delegate void MsgBoxCallback(HELPINFO lpHelpInfo);
+
+        public delegate bool MonitorEnumDelegate(IntPtr hMonitor, IntPtr hdcMonitor, ref RECT lprcMonitor, IntPtr dwData);
+
+        public delegate IntPtr WndProc(IntPtr hWnd, WindowMessage msg, IntPtr wParam, IntPtr lParam);
```

Comment:
No, the code generator doesn't touch delegates. Are you concerned that delegates are more complex if they take pointers than IntPtr?